### DISCUSSION TITLE
PS1 only one core"rearmed"?

### DISCUSSION THREAD

**Discussion by Linkinterteid** [OnionUI/Onion#1280] [General]
Where are the other emulator cores?  Standalone?  He supposedly exists, but he’s not there?

**Comment by schmurtzm** [comment]
What is "the other" ? What are you looking for ?
PCSX Standalone is not a core but a standalone emulator with better performances compared to the core version. It is located in Package Manager. (indeed it should be mentioned in the [PSX section](https://onionui.github.io/docs/emulators/psx) of the docs, I will update that it's already in my todo).
